index.html 

```diff
@@ -3,25 +3,33 @@
 	<title>Bubble Tea site</title>
 </head>
 <body>
-	<div class="topnav">
-		<img src="" alt="logo" style="width:600px; height:400px"/>
-		<a class="active" href=#home">Home</a>
-		<a href=#news">News</a>
-		<a href=#about">About</a>
-		<a href=#contact”>Contact</a>
-	</div
-	<div>
-		<img src="" alt="description of image">
-	</div>
+	<nav>
+		<div class="topnav">
+			<img src="" alt="logo" style="width:600px; height:400px"/>
+			<a class="active" href=#home">Home</a>
+			<a href=#news">News</a>
+			<a href=#about">About</a>
+			<a href=#contact”>Contact</a>
+		</div>
+	</nav>
+	<section>
```

Comment:
Wrap everything below `<nav>` in the new semantic tag `<main>`, but do not wrap `<footer>` in this as it's not part of the main content of the page.